### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by kerestan**

### CODE REVIEW

File: playbooks/10 - SP - Time Series Charts/{POST UPDATE] Trigger Updated Chart Queries.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Time Series Charts/{POST UPDATE] Trigger Updated Chart Queries.json]: name of this PB needs correction. Please follow standard guidelines or look for SFSP PB names as a reference

Comment (kerestan) [reply playbooks/10 - SP - Time Series Charts/{POST UPDATE] Trigger Updated Chart Queries.json]: I've updated the name of the post create and post update playbooks to more closely match those in SFSP

**Comment by kerestan** [comment]
To-Do list from today's meeting with Sujit:

- Add SOC SImulator and SOAF Framework as prerequisites (because of use of Scenario module)
- Rename "Time Series Chart Examples" to "Time Series Scenario Charts"
- Remove dash from "single-" in SP descriotion
- Add documentation link 
- Move chart playbooks into use case folder
- Rename scenario playbook "Scenario - Generate Alerts for last 30 Days"
- Add "scenario" tag to appropriate playbook
- Content hub json file needs to be changed. Line 442 change from "time-series-charts" to "timeSeriesCharts" to match widget name in info.json
- Scenario record needs to be changed to a "reset" state (it has record id's which. it should not)
- Scenario playbook generates an error on build 3008, need to update logic to not use
- Add snapshots to  widget info.json
- Add helpOnline field to widget info.json
- Widget documentation images not loading correctly. Use syntax Sujit provided
- Widget documentation showing extra contents
- Connector missing documentation AND help online link
- Need to add time series charts module in navigation and permissions for the module (add to  all default roles)
- Need to add records in time series charts module
- Add step  to scenario to run "update all time  series charts" playbook
- Make all module fields default grid  columns
- Add cleanup playbook's code snippet as operation into connector
- Add "On Create" and "On Update" and "Manual" tags to appropraite playbooks. In Manual trigger one remove  "manual" from name as well
- Set all playbooks to INFO mode
- Turn off inline editor in module detail view

**Comment by cs-sujeet-ahirrao** [comment]
Looks good to me

**Pull request merged**